cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think this file was re-added by accident (it was deleted in a previous PR).

Reply:
Hey Matt... I think this happens automatically when you run the test the first time ... I forgot to delete it